app/controllers/journals_controller.rb

```diff
@@ -110,17 +84,6 @@ def diff
     end
   end
 
-  def preview
-    @journal.notes = params[:notes]
-
-    respond_to do |format|
-      format.any(:html, :js) do
-        render locals: { journal: @journal },
-               layout: false
-      end
-    end
-  end
-
   private
 
   def find_journal
```

Comment:
Wait a sec, I overlooked that the `diff` method is still used. So you can remove less of what I wrote but you can still remove some of it.